packages/design/src/DataTable/Cells.tsx

```diff
@@ -82,12 +82,16 @@ export const DateCell = ({ data }: { data: Date }) => (
 
 const renderLabelCell = (labels: string[] = []) => {
   const $labels = labels.map(label => (
-    <Label mb="1" mr="1" key={label} kind="secondary">
+    <Label mr="1" key={label} kind="secondary">
       {label}
     </Label>
   ));
 
-  return <Cell>{$labels}</Cell>;
+  return (
+    <Cell>
+      <div style={{ lineHeight: '20px' }}>{$labels}</div>
```

Comment:
style uses inline styling, i think from prior discussions we preferred css
```suggestion
      <div css={{ lineHeight: '20px' }}>{$labels}</div>
```